src/ctap/storage.rs

```diff
@@ -1203,7 +1209,8 @@ mod test {
             persistent_store.set_min_pin_length_rp_ids(rp_ids.clone()),
             Ok(())
         );
-        for rp_id in DEFAULT_MIN_PIN_LENGTH_RP_IDS {
+        for rp_id in DEFAULT_MIN_PIN_LENGTH_RP_IDS.iter() {
+            let rp_id = rp_id.to_string().to_string();
```

Comment:
That's weird. You're sure you need both `to_string()`? What about `for &rp_id in ...`? Same for the loop above.

Reply:
Ah, I forgot to fix the test to match the implementation.